Dev-8/ElementOfMatrix.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Class to determenetion one elemnt of matrix.
+ */
+class ElementOfMatrix {
+  private double element = 0;
+
+/**
+ * Method determiny elment of matrix and return it.
+ */
+  public double get(double[][][] mass, int lineNumber, int rawNumber) {
```

Comment:
Too complicated, do not use one 3-dimensional massive instead of two 2-dimentional, no OOP